main.go

```diff
@@ -1177,21 +1170,21 @@ var importCmd = &cobra.Command{
 	RunE: func(cmd *cobra.Command, args []string) (err error) {
 		var in io.Reader
 		if len(args) > 0 {
-			var f *os.File
-			f, err = os.Open(args[0])
+			filename := args[0]
+			file, err := os.OpenFile(filename, os.O_RDONLY, 0600)
 			if err != nil {
 				return err
 			}
 			defer func() {
-				if errc := f.Close(); err == nil && errc != nil {
-					err = errc
+				if e := file.Close(); e != nil && err == nil {
+					err = e
 				}
 			}()
-			in = f
+			in = file
 		} else {
 			in = os.Stdin
 		}
-		if err = registry.Import(in); err != nil {
+		if err = registry.Import(in, importDrop); err != nil {
```

Comment:
[nit] I would have called `registry.Drop()` from this file, instead of passing the `importDrop` flag to `registry.Import()`